incydr/cli/cmds/cases.py

```diff
@@ -269,6 +269,10 @@ def bulk_update(ctx: Context, csv: Path):
             description="Updating cases...",
             transient=True,
         ):
+            if case.assignee:
+                case.assignee = user_lookup(client, case.assignee)
+            if case.subject:
+                case.subject = user_lookup(client, case.subject)
```

Comment:
For now I've just added username/userId validation here.  If we want to develop a different strategy for CSV processing we can change this with that ticket.